src/features/layout/NavbarItem.tsx

```diff
@@ -9,58 +9,65 @@ import { types } from './reducer'
 import classnames from 'classnames'
 import { Omit } from '@material-ui/types'
 
-const useStyles = makeStyles((theme: Theme) => ({
-    gutters: {
-        padding: `0.65em 0em`,
-    },
-    active: {
-        backgroundColor: theme.palette.background.default,
-            color: theme.palette.primary.main,
-            '& svg': {
-                color: theme.palette.primary.main,
-            },
-    },
-    root: {
-        marginLeft: '0.5em',
-        borderRadius: '3em 0 0 3em',
-        overflow: 'hidden',
-        '& svg': {
-            margin: 'auto',
+const useStyles = makeStyles((theme: Theme) => {
+    const itemColor =
+        theme.palette.type === 'light' ?
```

Comment:
looks not good. You should avoid logic which depends on theme name. Better to avoid any conditional logic here at all. If you need conditional logic it might mean that you have to add more colors into palette and/or better organize colors for your themes 

Just imagine that in the future you'll need to add one more theme or significantly change colors in existing themes. You'll have to go through all code with conditional logic in order to verify that it's still working properly

Reply:
have you fixed this? I don't see it's fixed

Reply:
I am sorry, I forgot push commit